code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -49,7 +49,7 @@
 					new /obj/item/weapon/gun/projectile/pistol(get_turf(H))
 					new /obj/item/gun_part/silencer(get_turf(H))
 				if("cannon")
-					new /obj/item/weapon/gun/energy/lasercannon(get_turf(H))
+					new /obj/item/weapon/gun/energy/laser/cannon(get_turf(H))
```

Comment:
He was told to get rid of copypasted code and did.

Stop trying to somehow disqualify the PR from getting merged just because the vote didn't go the way you wanted.